apps/OboeTester/app/src/main/java/com/mobileer/oboetester/GlitchActivity.java

```diff
@@ -175,6 +178,7 @@ private String getCurrentStatusReport() {
             message.append(String.format("glitch.frames = %d\n", mLastGlitchFrames));
             message.append(String.format("reset.count = %d\n", mLastResetCount - mStartResetCount));
             message.append(String.format("peak.amplitude = %8.6f\n", mPeakAmplitude));
+            message.append(String.format("sine.amplitude = %8.6f\n", mSineAmplitude));
```

Comment:
This overflows the line count for the TextView so the "glitch.count" line is now missing.

Reply:
Nice catch!